torch/_inductor/codegen/triton_utils.py

```diff
@@ -10,7 +12,14 @@ def signature_of(arg: Union[TensorArg, SizeArg], *, size_dtype: str) -> str:
     from triton.runtime.jit import JITFunction
 
     if isinstance(arg, TensorArg):
-        tye = JITFunction._type_of(arg.dtype)
+        # TODO: Remove fp8 special handling when Triton supports PyTorch fp8 dtypes.
+        # Related PR: https://github.com/openai/triton/pull/2279/
```

Comment:
Looks like this was merged but I imagine the triton version in pytorch hasn't been updated to have this commit

Reply:
Yeah I think we need to wait for Triton version update in Pytorch to remove these lines.